README.md

```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 1
+# Self-Introduction
+![](https://artificery.com/wp-content/uploads/2017/09/portland-oregon.jpg)
+
```

Comment:
Successful use of link syntax in Markdown to insert image of Portland. Nice overall aesthetic too - great job.